lib/matplotlib/cbook.py

```diff
@@ -1941,8 +1941,8 @@ def _bootstrap_median(data, N=5000):
         M = len(data)
         percentiles = [2.5, 97.5]
 
-        ii = np.random.randint(M, size=(N, M))
-        bsData = x[ii]
+        bs_index = np.random.randint(M, size=(N, M))
```

Comment:
The latter is not pep8 compliant, so I'd rather go for the pep8 compliant one though it is not coherent on this particular function.